test/datatype/Reduce_local_float.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
```

Comment:
I would like to see these 2 files merged into a single one that tests all ops and all datatypes.